app/views/components/_option-select.html.erb

```diff
@@ -30,6 +30,8 @@
 
 <div class="app-c-option-select <% if expander_style %>app-c-option-select--expander<% end %>"
   <% if local_assigns.include?(:closed_on_load) && closed_on_load %>data-closed-on-load="true"<% end %>
+  <% if local_assigns.include?(:closed_on_desktop) && closed_on_desktop %>data-closed-on-desktop="true"<% end %>
+  <% if local_assigns.include?(:closed_on_mobile) && closed_on_mobile %>data-closed-on-mobile="true"<% end %>
```

Comment:
Do we need to have separate values for desktop and mobile? Can `closed_on_load` expanded include the different conditions for desktop and mobile?